cmd/sonobuoy/app/featureGates.go

```diff
@@ -10,6 +10,18 @@ const (
 	FeatureWaitOutputProgressByDefault = "SONOBUOY_WAIT_PROGRESS"
 )
 
+var (
+	featureDefaultMap = map[string]bool{
+		FeaturePluginInstallation:          true,
+		FeatureWaitOutputProgressByDefault: true,
+	}
+)
+
 func featureEnabled(feature string) bool {
-	return os.Getenv(FeaturesAll) == "true" || os.Getenv(feature) == "true"
+	// Allow features we default as true to be turned off while still relatively new so if major
+	// bugs are found we have workarounds.
+	if os.Getenv(feature) == "false" {
+		return false
+	}
+	return featureDefaultMap[feature] || os.Getenv(FeaturesAll) == "true" || os.Getenv(feature) == "true"
```

Comment:
I'm going to add a small unit test for this. Super easy to test and without it super easy to miss one of the variations.